src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -118,7 +119,10 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
         .setOption('sortOrder', getLogsPanelSortOrder())
         .setOption('wrapLogMessage', Boolean(getLogOption<boolean>('wrapLogMessage', false)))
         .setOption('showLogContextToggle', true)
-
+        // @ts-expect-error
+        .setOption('enableInfiniteScrolling', true)
```

Comment:
can we comment what version of core is needed to remove these?

Reply:
Definitely. Done.